src/panel_gwalker/_pygwalker.py

```diff
@@ -1,6 +1,9 @@
-from typing import TYPE_CHECKING, Any, Dict, List
+import json
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Protocol
 
 import pandas as pd
+from sqlalchemy import create_engine, text
```

Comment:
I don't think this should be imported at the top-level if it can be avoided.

Reply:
I agree